### ISSUE TITLE
Cursor jumping

### ISSUE THREAD

**Issue by mlajtos** [andrejewski/srepl#1]
I ❤️ SREPL

Is there anything that can be done about jumping cursor?

https://user-images.githubusercontent.com/1047512/218335014-ab87f3dc-f66d-4b95-bd22-0e212901952d.mov

**Comment by andrejewski** [comment]
@mlajtos I assume you're using VSCode? That's what I use and I also experience the cursor jumping. Since this SREPL implementation doesn't directly integrate with the editor, we can only hope to write back the file in such a way cursor position is retained.

I haven't explored this at all, so there's a few things we could test out. Special line endings are one example

**Comment by mlajtos** [comment]
Yes, I am using VSCode. If SREPL would touch only that part of the line that needs to be updated, the cursor would be probably okay. Maybe via streams from `fs`?
